core/templates/pages/exploration-player-page/templates/information-card-modal.directive.html

```diff
@@ -56,10 +56,10 @@ <h2 ng-style="titleWrapper()" class="oppia-info-card-title"><[explorationTitle]>
     </ul>
 
     <div ng-if="!explorationIsPrivate" class="oppia-info-card-bottom-row">
-      <div flex="40">
+      <div flex="40" class="tag-spacing">
         <md-content layout="row">
           <div class="oppia-info-card-tag-icon">
-            <span class="fas fa-tags" tooltip-append-to-body="true"
+            <span class="fas fa-tags spaced" tooltip-append-to-body="true"
```

Comment:
Let's use `oppia-info-card-modal-tooltip` instead of `spaced`. spaced seems too generic.